debezium-server/debezium-server-redis/src/main/java/io/debezium/server/redis/RedisStreamChangeConsumer.java

```diff
@@ -51,10 +52,16 @@
     private static final String PROP_ADDRESS = PROP_PREFIX + "address";
     private static final String PROP_USER = PROP_PREFIX + "user";
     private static final String PROP_PASSWORD = PROP_PREFIX + "password";
+    private static final String PROP_MINID = PROP_PREFIX + "minid";
+    private static final String PROP_MAXLEN = PROP_PREFIX + "maxlen";
 
     private HostAndPort address;
     private Optional<String> user;
     private Optional<String> password;
+    private Optional<String> minIdTrim;
```

Comment:
Is there a specific reason why we cannot inject this via `@ConfigProperty`, like below?

Reply:
Bc this is an optional field.

Reply:
but i can use ConfigProperty if it looks nicer?

Reply:
It's not so much about looking nicer, more about just doing one thing (obtaining config properties) in one way, instead of two different ways, if possible.

Reply:
Once changing those to ConfigProperty, they should be documented in debezium-server.adoc.